validator/chromeextension/background.js

```diff
@@ -257,6 +281,44 @@ function validateUrlFromTab(tab) {
   xhr.send();
 }
 
+/**
+ * Event handler which gets called for onBeforeSendHeaders
+ * (developer.chrome.com/extensions/webRequest#event-onBeforeSendHeaders) and
+ * updates the User-Agent header to the value that's been specified. Handler listers
```

Comment:
Looks like this comment wasn't finished or it's some text that's left over: `Handler listers`